### PR TITLE
45677805 MRI super-resolution network SRCNN

### PR DESCRIPTION
**Pull request by leoUchiha**
Hi, i implemented SRCNN model for prooblem 5  MRI super-resolution network with any CNN model  on the ADNI brain dataset.
Due to limitation of GPU usage and time constraint, the result output is not very clear as from paper due to it only trained 265 epchos and variant 9-5-5-1 architecture has been tried but the result is not showing obvious improvment.
It contains following files:
> **README.md:** contains problems description it solved, algorithm description, how it works, the architecture of the SRCNN model, and input, out result and result comparion with simple tensorflow image resize function
> **dataset.py:**  functions that load the dataset and data preprocessing
> **modules.py:** the function that builds the SRCNN model
> **predict.py:**  functions that generate output images and processing the output images
> **train.py:** functions that compile the model and train the model with feed data, plot the loss graph
> ** dataset1.ipynb:** is full code working version on the local computer
> ** srcnn.ipynb:** is full code working version on colab which is the version i used to train the model
> **predict_result:** is the folder contains input, output images result taken either by plt.save function or by screenshot for down-scaling images due to plt will auto zoom in the images.
> **model:** contains trained model, accidentally uploaded
Chongyu Huang
45677805
Thanks

### CODE REVIEW

**Comment by uqmarlonbran** [comment]
Uses SRCNN rather than ESPCNN for super rez. Results ok, but slightly worse than ESPCNN implementations.

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage 
Module present
Commenting
No Data leakage
Difficulty: Normal -5
## Commit Log
Meaningful commit messages -1 (could be more descriptive)
Progressive commits used
## Documentation
ReadMe acceptable/good
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description good
